### DISCUSSION TITLE
2023/01/05/the-benchmark-hunting-section-of-geocaching-com-is-gone

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#21] [Announcements]
# 2023/01/05/the-benchmark-hunting-section-of-geocaching-com-is-gone

http://localhost:4000/2023/01/05/the-benchmark-hunting-section-of-geocaching-com-is-gone

**Comment by jengalas** [comment]
Posted by Mike Boyer
March 14, 2024 at 7:17 pm
_Imported from WordPress_

I have some PIDs that weren’t archived…😡😡😡

**Comment by jengalas** [comment]
Posted by David S.
February 27, 2024 at 8:59 pm
_Imported from WordPress_

Zhanna, I just discovered this travesty that you so eloquently speak of. I had migrated away from most Geocaches with the exception of some unique or interesting hides several years ago but my Benchmark finds were my favorites simply because some of them required digging with a shovel or clearing out shrubs and weeds to “unearth” that sweet disc! I had posted many pictures of my Benchmark finds but now that’s gone too. Damn shame. I also went out of my way to document destroyed marks for anyone using the database in hopes of finding them. I would go as far as scouring old topo maps and acquiring old aerial photos to cross check. I enjoyed this immensely. I would like to say thank you for your time and effort in documenting this and providing links to try and keep things going. The zip code search will be sorely missed though.

Take care!

Kind regards, David S.

**Comment by jengalas** [comment]
Posted by Brian Christian
February 4, 2024 at 4:56 pm
_Imported from WordPress_

I too am Quite Disappointed with Groundspeak for behaving like Google/Alphabet. As a geocacher with a strong interest in GIS and cartography, I quite enjoyed the benchmark finding section, and thought perhaps (oh no, a public service) it fed into the NOAA database(s). Perhaps keeping track of the APIs became too onerous, but I doubt it — and agree with your conclusions.

n.b. the geodesy dataset/input appears to have moved here: https://geodesy.noaa.gov/surveys/mark-recovery/index.shtml

**Reply by jengalas** [reply]
Posted by Zhanna
February 4, 2024 at 8:18 pm
_Imported from WordPress_

Thank you, Brian! Over a year later, this decision still stings.

**Comment by jengalas** [comment]
Posted by Tom Casey
January 27, 2023 at 4:09 pm
_Imported from WordPress_

As Linn County, (Oregon) Surveyor, I used the Benchmark Hunting Section on many searchs for benchmarks with much success. To see this section “Retired” is quite a blow.
